eve-api-data/src/test/resources/application.yml

```diff
@@ -0,0 +1,15 @@
+spring:
```

Comment:
This file is picked up by default as the application.yml or application.properties file are the heart of configuring A LOT of things in a spring-boot world!